### PR TITLE
[RAPTOR-11619] update public environments with new base images

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Update public envs with new base images, which have env var PYTHONUNBUFFERED=1 set. Which should help with printing user messages.

I'll install new envs on staging once PR is merged.

### CODE REVIEW

File: docker/dropin_env_base_jdk_ubi/build_image.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ IMAGE_ORG_DATAROBOTDEV=datarobotdev
 
 IMAGE_REPO=dropin-env-base-jdk
 
-IMAGE_TAG=ubi8.8-py3.11-jdk11.0.22-drum1.13.0-mlops9.2.8
+IMAGE_TAG=ubi8.8-py3.11-jdk11.0.25-drum1.13.0-mlops9.2.8
```
Comment (yakov-g) [inline docker/dropin_env_base_jdk_ubi/build_image.sh:29]: This is a residue left from a prev PR. Fix it ti have code updated.

Doesn't really matter whether tag has `22` or `25` . Image already pushed so I'll use tag with `22`

**Review by klichukb — APPROVED** [review]
@ben-cutler-datarobot @yakov-g thank you both, this is a great improvement.

**Review by elatt — APPROVED** [review]

**Review by jpclemens0 — APPROVED** [review]
LGTM for GenAI

**Review by baekdahl — APPROVED** [review]
@yakov-g I would like to release a new version of DRUM as soon as possible. Should we coordinate that with this PR so we only need to build and install environments once?

**Comment by yakov-g** [comment]
@baekdahl 
I can wait, release please.

**Comment by baekdahl** [comment]
@yakov-g https://github.com/datarobot/datarobot-user-models/pull/1167

**Comment by yakov-g** [comment]
I need to investigate why fit fails for keras env now. I think I know the direction, but don't know the actual reason.

**Pull request merged**